scan/src/components/data-source/DataSourceExecute.re

```diff
@@ -86,11 +86,17 @@ let parameterInput = (name, placeholder, index, setCalldataArr) => {
   </div>;
 };
 
+type result_data_t = {
+  returncode: int,
+  stdout: string,
+  stderr: string,
+};
+
```

Comment:
@sorawit Why don't we use a camel case?

Reply:
It's one word: `returncode` like calldata.